ome_zarr.py

```diff
@@ -169,3 +171,16 @@ def get_json(self, subpath):
         except:
             print("FIXME", rsp.text, dir(rsp))
             return {}
+
+
+def info(path):
+    """
+    print information about the ome-zarr fileset
+    """
+    zarr = parse_url(path)
+    if not zarr.is_ome_zarr():
+        print("not an ome-zarr: {zarr}")
```

Comment:
```suggestion
        print(f"not an ome-zarr: {zarr}")
```

Reply:
I think you want ```print(f"not an ome-zarr: {path}")``` ?

Reply:
I was thinking that thing would eventually have a nice `__str__` to explain _why_ it wasn't an ome-zarr.